tests/instances.spec.ts

```diff
@@ -247,3 +247,53 @@ test("Create an image from an instance", async ({ page }) => {
   const imageName = await getImageNameFromAlias(page, imageAlias);
   await deleteImage(page, imageName);
 });
+
+test("Bulk start, pause, unpause and stop instances", async ({ page }) => {
+  await page.goto("/ui/");
+  await page.getByPlaceholder("Search").click();
+  await page.getByRole("button", { name: ": Container" }).click();
+
+  //Bulk start instances
+  await page
+    .getByRole("row", { name: "select Name Type Description Status Actions" })
+    .getByLabel("Select all")
+    .click();
+  await page.getByRole("button", { name: "Start", exact: true }).click();
+  await page
+    .getByLabel("Confirm start")
+    .getByRole("button", { name: "Start" })
+    .click();
+  await page.waitForSelector(`text=instances started.`);
```

Comment:
```suggestion
  await page.waitForSelector(`text=instance started.`);
```
Now it is a singular case, as we filter the other vm instance. Same for the other tests below.

Reply:
Maybe we should search for the container instance from the test suite instead of filtering by container type. So this also passes locally with a busy default project containing many containers.

Reply:
I am not sure I understand. Are you saying we should put the two tests in its own suite and do the filter then, what would this achieve?
The tests pass locally for me when adjacent with other non related instances I have.

Reply:
Or perhaps are you saying that the test should be conducted in a project of it's own so that it is created, empty?

Reply:
Right, I believe I understand now. I'll get that done.